### PR TITLE
upgrade zookeeper and kafka images

### PR DESCRIPTION
**Pull request by wind57**

### CODE REVIEW

**Comment by wind57** [comment]
while debugging that `ActuatorRefreshKafkaIT` problem, I've noticed that we are using some really old images for zookeeper and kafka. This PR upgrades those. Of course the build success is dependent on the fix of the test

File: spring-cloud-kubernetes-integration-tests/spring-cloud-kubernetes-configuration-watcher-it/src/test/resources/kafka/kafka-deployment.yaml [hunk 1]
```diff
@@ -17,19 +17,41 @@ spec:
...
+
+            - name: KAFKA_ADVERTISED_HOST_NAME
+              valueFrom:
+                fieldRef:
+                  fieldPath: status.podIP
+
+            - name: KAFKA_ZOOKEEPER_CONNECT
+              value: zookeeper:2181
+
+            # we have enabled auto creation of topics and when this happens there is a replication factor of 3
```
Comment (wind57) [inline spring-cloud-kubernetes-integration-tests/spring-cloud-kubernetes-configuration-watcher-it/src/test/resources/kafka/kafka-deployment.yaml:53]: this is one more important setting

**Comment by wind57** [comment]
@ryanjbaxter would you take a look at this one also please?

**Comment by ryanjbaxter** [comment]
Is there a reason why you are switching the images?

**Comment by ryanjbaxter** [comment]
Ah these are the official community images?

**Comment by wind57** [comment]
Nope, not really. 

Its that these are deprecated for a very long time and are non-oficial ones. I saw the opportunity to fix this.

**Pull request merged**